superset/assets/javascripts/explorev2/components/SelectField.jsx

```diff
@@ -42,8 +42,18 @@ export default class SelectField extends React.Component {
     if (this.props.freeForm) {
       // For FreeFormSelect, insert value into options if not exist
       const values = this.props.choices.map((c) => c[0]);
-      if (values.indexOf(this.props.value) === -1) {
-        options.push({ value: this.props.value, label: this.props.value });
+      if (this.props.value) {
```

Comment:
might be cleaner to pull this out into another method and add a comment about how this is for supporting the free form select field options.